sirch-inc/sirch-coin
# SIRCO-91: Transfer History Page

This update adds a users transfer history to the “Transfers” page. It also creates a new reusable component, TransferCard, which is used to display both Sent and Received Transfers. 

Additional notes:
* Adds some temporary desktop-only styling to the transfers page for now, for clearer review purposes
* Transfers are separated between Sent and Received Transfers - open to alternative solutions down the line
* These modules were previously called “Transactions”, but have been reverted back to “Transfers”. At some point in the future, will want to change back to “Transactions” to show not only transfers between users, but also transactions of using Sirch Coins to businesses/buying stuff.
* Installed a date formatter, date-fns, to format Supabase dates as they come in. Can be reused on other components as needed in the future. 

To test both Sent and Received Transfers:
* Login with one account and send some coins to another one of your test accounts 
* Login with another test account and send some coins to the account from step 1 
* Log back into the account from step 1 and visit the transfers page to ensure accuracy of all transfers that have occurred.
